Sprint-3/4-stretch-investigate/card-validator.js

```diff
@@ -0,0 +1,55 @@
+function cardNumberValidator(cardNumber) {
+  if (cardNumber.length !== 16) {
+    return false;
+  }
+  const numberInArray = cardNumber.toString().split(""); // convert the cardNumber to sting then into an array
```

Comment:
It's interesting that you check its length and _then_ convert it to a string. What type do you think `cardNumber` may have which has a `.length` property, but isn't a string?